Source/Node/src/MFRC522Reader.cpp

```diff
@@ -0,0 +1,112 @@
+#include "MFRC522Reader.hpp"
+#include <SPI.h>
+
+
+#define DEFAULT_SS_PIN 10
+#define DEFAULT_RST_PIN 9
+#define DEFAULT_KEY 0xFF
+#define DEFAULT_TRAILER_BLOCK 7
+
+MFRC522Reader::MFRC522Reader()
+{
+    reader = MFRC522(DEFAULT_SS_PIN, DEFAULT_RST_PIN);
```

Comment:
What if it does not init correctly? I dont see error handling. Is this not possible?

Reply:
cant use error handling because of void type initializers